src/common/components/header.tsx

```diff
@@ -7,15 +7,22 @@ import { NamedFC } from 'common/react/named-fc';
 import { title } from 'content/strings/application';
 import * as styles from './header.scss';
 
-export type HeaderProps = { deps: HeaderDeps };
 export type HeaderDeps = HeaderIconDeps;
 
+export type HeaderProps = {
+    deps: HeaderDeps;
+    nearItems?: JSX.Element;
```

Comment:
The names near and far didn't make sense to me at first, but I'm not sure if names like 'leftItems' would be better or if they're too specific

Reply:
So `nearItems` and `farItems` follow the naming choice of Office UI Fabric [CommandBar](https://developer.microsoft.com/en-us/fabric#/controls/web/commandbar). I think they avoid using `leftItems` and `rightItems` to account for RTL languages where using left and right to name items and mention how they are positioned in the general UI layout is not correct. 
I could change the name but I agree with you: left and right doesn't make more/better sense to me.

Reply:
`leftItems`/`rightItems` would assume a specific reading direction, which isn't true for all languages (though for now that's not an issue for AI-Web). I do note that the Office Fabric [CommandBar](https://developer.microsoft.com/en-us/fabric#/controls/web/commandbar) uses `items` and `farItems`; would it make sense to be consistent with this? Or is there other precedent for `nearItems`?

Reply:
I swear I saw `nearItems`... I'll update to use `items` and `farItems`.

Reply:
Updated